helmfile.d/stacks/gpu-operator.yaml.gotmpl

```diff
@@ -0,0 +1,12 @@
+---
+templates:
+  nvidia-gpu-operator:
```

Comment:
Question: This is probably me not remembering how this stacking works. But where do you specify that it's suppose to use the `upstream/nvidia/gpu-operator` chart?

Reply:
[This file](https://github.com/elastisys/compliantkubernetes-apps/blob/2fda3c8587bb2140f9d083ffa914e657794b7d5c/helmfile.d/bases/upstream.yaml.gotmpl) generates templates for all upstream charts in the `upstream/index.yaml` file and then that template [is referred to here](https://github.com/elastisys/compliantkubernetes-apps/blob/2fda3c8587bb2140f9d083ffa914e657794b7d5c/helmfile.d/stacks/gpu-operator.yaml.gotmpl#L4).